config/tgl-cavs.toml

```diff
@@ -0,0 +1,196 @@
+version = [2, 6]
+
+[adsp]
+name = "tgl"
+machine_id = 10
+image_size = "0x100000"
+
+[[adsp.mem_zone]]
+type = "ROM"
+base = "0x9F180000"
+size = "0x00002000"
+[[adsp.mem_zone]]
+type = "IMR"
+base = "0xB0000000"
+size = "0x1000000"
+[[adsp.mem_zone]]
+type = "HP-SRAM"
+base = "0xBE000000"
+size = "0x800000"
+[[adsp.mem_zone]]
+type = "LP-SRAM"
+base = "0xBE800000"
+size = "0x40"
+
+[cse]
+partition_name = "ADSP"
+[[cse.entry]]
+name = "ADSP.man"
+offset = "0x5c"
+length = "0x464"
+[[cse.entry]]
+name = "cavs0015.met"
+offset = "0x4c0"
+length = "0x70"
+[[cse.entry]]
+name = "cavs0015"
+offset = "0x540"
+length = "0x0"  # calculated by rimage
+
+[css]
+
+[signed_pkg]
+name = "ADSP"
+[[signed_pkg.module]]
+name = "cavs0015.met"
+
+[adsp_file]
+[[adsp_file.comp]]
+base_offset = "0x2000"
+
+[fw_desc.header]
+name = "ADSPFW"
+load_offset = "0x30000"
+
+[module]
+count = 7
+	[[module.entry]]
+	name = "BRNGUP"
+	uuid = "61EB0CB9-34D8-4F59-A21D-04C54C21D3A4"
+	affinity_mask = "0x1"
+	instance_count = "1"
+	domain_types = "0"
+	load_type = "1"
+	module_type = "0"
+	auto_start = "0"
+
+	[[module.entry]]
+	name = "BASEFW"
+	uuid = "383B9BE2-3518-4DB0-8891-B1470A8914F8"
+	affinity_mask = "3"
+	instance_count = "1"
+	domain_types = "0"
+	load_type = "1"
+	module_type = "0"
+	auto_start = "0"
+
+	[[module.entry]]
+	name = "MIXIN"
+	uuid = "39656EB2-3B71-4049-8D3F-F92CD5C43C09"
+	affinity_mask = "0x1"
+	instance_count = "30"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "1"
+	auto_start = "0"
+	sched_caps = [1, 0x00008000]
+	pin = [0, 0, 0xfeef, 0xc, 0x8, 0x45ff,
+		   1, 0, 0xfeef, 0xc, 0x8, 0x1ff,
+		   1, 0, 0xfeef, 0xc, 0x8, 0x1ff,
+		   1, 0, 0xfeef, 0xc, 0x8, 0x1ff,
+		   1, 0, 0xfeef, 0xc, 0x8, 0x1ff]
+	mod_cfg = [ 0, 0, 0, 0, 296, 644000, 45, 60, 0, 0, 0,
```

Comment:
ditto for mod_cfg - The intention is that anyone should be able to understand these and edit them (until automatic generation is ready).